adlik_serving/runtime/tensorrt/model/trt_instance.cc

```diff
@@ -216,8 +175,8 @@ tensorflow::Status Instance::loadPlan(const std::vector<char>& model_data) {
 
 tensorflow::Status Instance::allocBuffer() {
   const int num_expected_bindings = engine->getNbBindings();
-  byte_sizes = new uint64_t[num_expected_bindings];
-  buffers = new void*[num_expected_bindings]();
+  byte_sizes.reset(new uint64_t[num_expected_bindings]);
```

Comment:
Better to use `std::make_unique` to create array and avoid using raw `new` operator.